Adds Docker image support and ability to specify custom boards (#38)
antmicro/renode-linux-runner-action: README.md

### OS configuration

- [`rootfs-size`](#rootfs-size) - Set size of the rootfs image. Default: auto

> - [`image-type`](#image) - native or docker. Read about the differences in the [image section](#image)
> - [`image`](#image) - URL of the path to tar.xz archive with linux rootfs for the specified architecture or docker image identifier. If not specified, the action will use the default one. See releases for examples.

Reviewer: ```suggestion
- [`image-type`](#image) - `native` or `docker`. Read about the differences in the [Image section](#image)
- [`image`](#image) - URL of the path to `tar.xz` archive with Linux rootfs for the specified architecture or Docker image identifier. If not specified, the action will use the default one. See releases for examples.
```

### Borad and devices configuration

- [`network`](#network) - Turn on the Internet in the emulated Linux? Default: true
- [`devices`](#devices) - List of devices to add to the workflow. If not specified, the action will not install any devices.
- [`kernel`](#kernel) - URL of the path to the tar.xz archive containing the compiled embedded linux kernel + initramfs. If not specified, the action will use the default. See releases for examples.
- [`arch`](#emulation) - Processor architecture

> - [`board`](#board) - Board name or `default` for architecture default, or `custom` for custom board

Reviewer: ```suggestion
- [`board`](#board) - A specific board name, or `default` for architecture default, or `custom` for a custom board
```

## Kernel

> This action offers to replace the Linux image on which the tests are run and mount the [file system image](#image). More information on how to do it [here](docs/Kernel.md).

Reviewer: ```suggestion
It is possible to replace the Linux image on which the tests are run and mount the [file system image](#image). More information on how to do it can be found [in the 'Kernel' section of the docs](docs/Kernel.md).
```
